jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -101,6 +101,39 @@ jobs:
        BAR: bar
 ```
 
+### Modifying `$PATH`
+
+**1.0**
+
+```YAML
+machine:
+  environment:
+    PATH: "/path/to/foo/bin:$PATH"
+```
+
+**2.0**
+
+In 2.0, interpolating variables into the environment is not yet supported.
+
+As a workaround, you have to add it to your `.bashrc` then make sure your command shell loads it. We can also make sure it is loaded automatically by changing `$BASH_ENV` like so.
+
+```YAML
+version: 2
+jobs:
+  build:
+    working_directory: /tmp
+    docker:
+      - image: buildpack-deps:jessie
+    environment:
+       BASH_ENV: ~/.bashrc
```

Comment:
Not sure. It's tmp? In the above example, the one in `/tmp` has some random ID generated in the file name. The user would have to find that and add to it versus something they could include in their repo?